### ISSUE TITLE
Access to MarkdownParserContext from IMarkdownExtension.PipelineOnDocumentProcessed()?

### ISSUE THREAD

**Issue by pm64** [xoofx/markdig#670]
Is is possible to access, or arrange things so that access is possible to MarkdownParserContext from PipelineOnDocumentProcessed()? I'm not seeing any obvious way, perhaps I'm missing something.

**Comment by xoofx** [comment]
No, but as `DocumentProcessed` is at the very end of the Markdown Parser method, you can add a call yourself in-between the call to the parser and the renderer.
